### ISSUE TITLE
Connections tab: Error: Error creating WebGL context.

### ISSUE THREAD

**Issue by 42** [ipfs-shipyard/ipfs-webui#76]
Fresh ipfs node, open local webui, go to the connections tab
Chromium 43.0.2357.130 Ubuntu 14.04 (64-bit)

```
Error: Error creating WebGL context.
    at Object.THREE.ExceptionErrorHandler (http://localhost:5001/ipfs/QmS2HL9v5YeKgQkkWMvs1EMnFtUowTEdFfSSeMT4pos1e6/bundle.js:70395:14)
    at initGL (http://localhost:5001/ipfs/QmS2HL9v5YeKgQkkWMvs1EMnFtUowTEdFfSSeMT4pos1e6/bundle.js:99169:11)
    at new THREE.WebGLRenderer (http://localhost:5001/ipfs/QmS2HL9v5YeKgQkkWMvs1EMnFtUowTEdFfSSeMT4pos1e6/bundle.js:92710:2)
    at init (http://localhost:5001/ipfs/QmS2HL9v5YeKgQkkWMvs1EMnFtUowTEdFfSSeMT4pos1e6/bundle.js:321:16)
    at new module.exports (http://localhost:5001/ipfs/QmS2HL9v5YeKgQkkWMvs1EMnFtUowTEdFfSSeMT4pos1e6/bundle.js:588:3)
    at React.createClass.createGlobe (http://localhost:5001/ipfs/QmS2HL9v5YeKgQkkWMvs1EMnFtUowTEdFfSSeMT4pos1e6/bundle.js:1949:18)
    at React.createClass.componentDidMount (http://localhost:5001/ipfs/QmS2HL9v5YeKgQkkWMvs1EMnFtUowTEdFfSSeMT4pos1e6/bundle.js:1915:10)
    at chainedFunction [as componentDidMount] (http://localhost:5001/ipfs/QmS2HL9v5YeKgQkkWMvs1EMnFtUowTEdFfSSeMT4pos1e6/bundle.js:54587:9)
    at CallbackQueue.assign.notifyAll (http://localhost:5001/ipfs/QmS2HL9v5YeKgQkkWMvs1EMnFtUowTEdFfSSeMT4pos1e6/bundle.js:49574:22)
    at ReactReconcileTransaction.ON_DOM_READY_QUEUEING.close (http://localhost:5001/ipfs/QmS2HL9v5YeKgQkkWMvs1EMnFtUowTEdFfSSeMT4pos1e6/bundle.js:63234:26)THREE.ExceptionErrorHandler @ bundle.js:70393initGL @ bundle.js:99175THREE.WebGLRenderer @ bundle.js:92710init @ bundle.js:321module.exports @ bundle.js:588React.createClass.createGlobe @ bundle.js:1949React.createClass.componentDidMount @ bundle.js:1915chainedFunction @ bundle.js:54587assign.notifyAll @ bundle.js:49574ON_DOM_READY_QUEUEING.close @ bundle.js:63234Mixin.closeAll @ bundle.js:67128Mixin.perform @ bundle.js:67069batchedMountComponentIntoNode @ bundle.js:61179Mixin.perform @ bundle.js:67055ReactDefaultBatchingStrategy.batchedUpdates @ bundle.js:58233batchedUpdates @ bundle.js:65226ReactMount._renderNewRootComponent @ bundle.js:61314ReactPerf.measure.wrapper @ bundle.js:62553ReactMount.render @ bundle.js:61403ReactPerf.measure.wrapper @ bundle.js:62553(anonymous function) @ bundle.js:19React.createClass.statics.run.dispatchHandler @ bundle.js:47391(anonymous function) @ bundle.js:47359(anonymous function) @ bundle.js:46413(anonymous function) @ bundle.js:46413Transition.to @ bundle.js:46416(anonymous function) @ bundle.js:47358Transition.from @ bundle.js:46395dispatch @ bundle.js:47355refresh @ bundle.js:47406run @ bundle.js:47402runRouter @ bundle.js:48094(anonymous function) @ bundle.js:18jQuery.Callbacks.fire @ bundle.js:27053jQuery.Callbacks.self.fireWith @ bundle.js:27165jQuery.extend.ready @ bundle.js:27371completed @ bundle.js:27387
```

Also shows `CONNECTED TO 0 PEERS` despite 

```
$ ipfs swarm peers | wc --lines
20
```

**Comment by jbenet** [comment]
I'm not sure where this is coming from. Is there a compat issue with WebGL and Ubuntu Chrome?

**Comment by cryptix** [comment]
Despite the concrete reason, I wondered why that error stops execution of the whole UI. Shouldn't it still list the peers even if it can't draw the globe? (Think IE6 or ancient FF)

**Comment by waynenilsen** [comment]
I have the same issue

**Comment by loadletter** [comment]
Same issue on a PC with an old video card that can't run WebGL, it would be nice if the list could still be viewed

**Comment by dignifiedquire** [comment]
This will be fixed in #229

**Comment by hacdias** [comment]
As version 2.0.0 approaches its release, this issue doesn't seem relevant.
